code/modules/assembly/flash.dm

```diff
@@ -114,7 +114,7 @@
 		var/mob/living/silicon/robot/R = M
 		add_logs(user, R, "flashed", src)
 		update_icon(1)
-		M.Weaken(6)
+		M.Weaken(2)

```

Comment:
probably but the only people that don't have alternative stuns as apart of their arsenal is clockcult now [ok and gangs] which has a variety of ways to stun [flash > judi glasses > flash > get forever stun].

If you check to see what it was previously hmu so I can compare between this one and that one and make a "logical conclusion".